commune_app/all_models/communes.py

```diff
@@ -16,6 +18,7 @@ class Commune(models.Model):
     name = models.CharField(max_length=100, unique=True, blank=False)
     description = models.CharField(max_length=250, blank=True)
     wallet = models.IntegerField(default=0, validators=[validate_wallet])
+    founder = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, default=346)  # 346 = Admin
```

Comment:
Here you added one more class member(DB column) to the Commune model 
but I can't see where you migrate it.

btw. 
do we really need it? 
maybe we can keep up without this field?

Reply:
I have the migrations on the branch, I will add it 

I added it in order to be able to verify that if someone is removing people from the commune, it is only the person who made the commune. 

Is there a different way of verifying that without this field?

Reply:
I have the migrations on the branch, I will add it

I added it in order to be able to verify that if someone is removing people from the commune, it is only the person who made the commune.

Is there a different way of verifying that without this field?

Reply:
in the built-in auth-user we have the field is_superuser
I thought maybe we can use it for this purpose